### DISCUSSION TITLE
ctf_web/php_unserialize/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#29] [Show and tell]
# ctf_web/php_unserialize/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://ctf.tj.cn/ctf_web/php_unserialize/

**Comment by pdsir** [comment]
姬哥,加油,很不错啊

**Comment by Abyssun** [comment]
为什么上一篇XSS没有评论区😫😭

**Reply by ProbiusOfficial** [reply]
应该是忘了写开启评论的code了（x
